lib/utils/noFilesFoundError.js

```diff
@@ -11,9 +13,36 @@ class NoFilesFoundError extends Error {
 			fileList = [fileList];
 		}
 
-		const pattern = fileList.filter((i) => !i.startsWith('!')).join(', ');
+		const filteredFileList = fileList.filter((i) => !i.startsWith('!'));
+		const pattern = filteredFileList.join(', ');
 
 		this.message = `No files matching the pattern "${pattern}" were found.`;
+
+		if (
+			os.platform() === 'win32' &&
+			filteredFileList.filter((file) => file.match(/^'.*'$/)).length > 0
+		) {
+			const packageScript = process.env.npm_lifecycle_event || '';
+			const scriptContents = process.env.npm_lifecycle_script || '';
```

Comment:
Bad idea use this env variables

Reply:
Ah, this was just a 'nice to have' based on the comments on the original ticket. I was interested in trying to solve this in the 'nicest' way for the end user. I  found the `lifecycle_event` documented at: https://docs.npmjs.com/misc/scripts#current-lifecycle-event but noticed the `lifecycle_script` var in investigation. I originally used dummy data, but it could be less useful. Best to replace with just a static example in your opinion?

Reply:
Let's use dummy example here to avoid this problematic pattern e.g. 

"Please use escaped double quotes for a file pattern in your package.json e.g. `stylelint \"./file.css\"`

Reply:
Updated, thanks for the feedback both!